pkg/scheduler/eventhandlers_test.go

```diff
@@ -334,3 +342,81 @@ func TestPreCheckForNode(t *testing.T) {
 		})
 	}
 }
+
+func TestAddAllEventHandlers(t *testing.T) {
```

Comment:
This is not exactly testing everything about the method. Add a comment describing what this is testing.

Reply:
ok